vsandroid_vcxproj.lua

```diff
@@ -51,7 +51,13 @@
 		_p(2, "<RootNamespace>%s</RootNamespace>", cfg.project.name)
 		_p(2, "<MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>")
 		_p(2, "<ApplicationType>Android</ApplicationType>")
-		_p(2, "<ApplicationTypeRevision>1.0</ApplicationTypeRevision>")
+		if _ACTION >= "vs2017" then
```

Comment:
This block is kind of just wrong. There's three cases, but only two versions of VS support this. VS2015 supports both v1 and v2, and I'm fairly certain that VS2017 supports v2 - but I can't verify that. I think the solution I have in my PR is the way this should be handled, the user deciding what version they're going to use. When v4 comes out, I highly doubt anything except the compiler versions will change, perhaps some additional options but nothing *should* break. I'm happy for your defaults below to be used in the event the user doesn't provide the version, but I think the user should be able to control what this module does, especially since VS2015 supports two versions.

Thoughts?

Reply:
Push your amendment on top?

Reply:
> Push your amendment on top?

Happy for this to go through as-is, when I get a chance to sit down and apply my changes on top, I'll open another PR. :+1: